### PR TITLE
Add migration testing

### PR DESCRIPTION
**Pull request by yakky**
How about opt-in migrations in test / server commands?
This will enable travis to run migrations to catch errors like the ones you fixed in this branch

### CODE REVIEW

File: develop.py [hunk 1]
```diff
@@ -154,6 +159,7 @@ def shell():
     default_name = ':memory:' if args['test'] else 'local.sqlite'
 
     db_url = os.environ.get("DATABASE_URL", "sqlite://localhost/%s" % default_name)
+    do_migrations = args.get('--do-migrations', False) or os.environ.get("CMS_DO_MIGRATIONS", False)
```
Comment (ojii) [inline develop.py]: why should this argument, unlike all other arguments, have a env variable too? The goal of this branch is to make things easier and one way of doing that is by providing a single way to do things.

File: develop.py [hunk 2]
```diff
@@ -33,17 +33,22 @@
     -h --help                   Show this screen.
     --version                   Show version.
     --parallel                  Run tests in parallel.
+    --do-migrations             Use south migrations in test or server command.
```
Comment (ojii) [inline develop.py]: tiny nitpick, I'd prefer `--migrate`.

Comment (yakky) [inline develop.py]: Sure

File: develop.py (hunk 1 above)
Comment (yakky) [inline develop.py]: Agree, it was a leftover from original implementation.
Removed

File: .travis.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ install:
     - pip install -q -r "test_requirements/django-$DJANGO.txt" --use-mirrors
     - if [ $DATABASE_URL == 'mysql://root@127.0.0.1/djangocms_test' ]; then pip install -q mysql-python --use-mirrors; fi
 script:
-    python develop.py test
+    python develop.py test --do-migrations
```
Comment (ojii) [inline .travis.yml]: you forgot to change the argument here

Comment (yakky) [inline .travis.yml]: sorry, I should ban sunday morning coding from my schedule

**Pull request merged**